### PR TITLE
Add Vultr to default datasource_list.

### PR DESCRIPTION
**Pull request by smoser**
Vultr support was added under #827.  This enables it by default
in the debian package.

### CODE REVIEW

**Comment by smoser** [comment]
@ddymko see above.

**Comment by ddymko** [comment]
@smoser Thanks for letting me know

**Review by OddBloke — APPROVED** [review]
Thanks!

File: debian/po/templates.pot [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: cloud-init\n"
 "Report-Msgid-Bugs-To: cloud-init@packages.debian.org\n"
-"POT-Creation-Date: 2021-03-22 14:08-0500\n"
+"POT-Creation-Date: 2021-04-20 10:14-0400\n"
```
Comment (TheRealFalcon) [inline debian/po/templates.pot:11]: Why do we change this?

Comment (smoser) [reply debian/po/templates.pot:11]: I'm not really sure. I was following this commit.
e3e36b2870b70dc588c87ec492cf18a54f46395f

**Pull request merged**